openwisp-config/files/openwisp.agent

```diff
@@ -121,6 +121,14 @@ while [ -n "$1" ]; do
 			export POST_REGISTRATION_HOOK="$2"
 			shift
 			;;
+		--checksum_max_retries)
+			export CHECKSUM_MAX_RETRIES="$2"
+			shift
+			;;
+		--checksum_retry_delay)
```

Comment:
for consistency with the parameters above, this should be `checksum-retry-delay`